src/components/Table/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, memo } from "react";
+
+import Header from "./Header";
+import Rows from "./Rows";
+import "./Table.style.scss";
+import { IEvent, IProps } from "./Table.types";
+
+const Table = (props: IProps) => {
+  const [sort, setSort] = useState("");
+  const [reload, setReload] = useState(false);
+  const { data } = props;
+
+  function compareNumbers(a: any, b: any, columnId: string) {
```

Comment:
Here you are using `function`, in another place you declare method by arrow function, please use one convention